### PR TITLE
fix(sorbet): turn off runtime checks properly

### PR DESCRIPTION
**Pull request by peterdeme**
Fixing https://github.com/GetStream/stream-chat-ruby/issues/97

### CODE REVIEW

**Review by ferhatelmas — DISMISSED** [review]
LGTM

**Review by ferhatelmas — APPROVED** [review]

File: lib/stream-chat/client.rb [hunk 1]
```diff
@@ -74,18 +73,18 @@ def initialize(api_key, api_secret, timeout = nil, **options)
...
+    T::Sig::WithoutRuntime.sig { params(options: T.untyped).returns(Client) }
     def self.from_env(**options)
-      Client.new(T.must(ENV['STREAM_KEY']),
-                 T.must(ENV['STREAM_SECRET']),
-                 ENV['STREAM_CHAT_TIMEOUT'],
-                 **{ base_url: ENV['STREAM_CHAT_URL'] }.merge(options))
+      Client.new(ENV.fetch('STREAM_KEY'),
+                 ENV.fetch('STREAM_SECRET'),
+                 ENV.fetch('STREAM_CHAT_TIMEOUT', DEFAULT_TIMEOUT),
+                 **{ base_url: ENV.fetch('STREAM_CHAT_URL', DEFAULT_BASE_URL) }.merge(options))
```
Comment (peterdeme) [inline lib/stream-chat/client.rb:81]: Rubocop fix

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -23,6 +23,7 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
+          cache-version: 1
```
Comment (ferhatelmas) [inline .github/workflows/ci.yml:26]: why do we need this?

File: lib/stream-chat/client.rb (hunk 1 above)
Comment (ferhatelmas) [reply lib/stream-chat/client.rb:81]: looks fine

File: .github/workflows/ci.yml (hunk 2 above)
Comment (peterdeme) [reply .github/workflows/ci.yml]: previous attempt to fix the build :)

**Review by ferhatelmas — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (ferhatelmas) [reply .github/workflows/ci.yml]: reverted so good 👍🏻

**Pull request merged**